frontend/src/components/ReviewReminders.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="alert alert-danger mb-4" role="alert">
+    <p>
+      <i class="fas fa-exclamation-triangle"></i> Peer reviews must be requested on
+      <strong>unpublished model releases</strong> so that you can continue to revise your submission
+      based on reviewer guidance.
+    </p>
+    <p>
+      If you have already published your model
+      <strong>a new draft release will be created</strong> as an exact copy of your most recent
+      published release. If your model successfully passes peer review you can then choose to
+      publish the reviewed release as the latest version.
```

Comment:
Should we add a sentence that emphasizes that they will be able to change the version as they publish their model?

Reply:
Yeah good idea, will do. I'm also going to see if I can do anything to reduce the whole thing's wall-of-text-ness while I'm at it